learn-arai/app/components/icons/line.tsx

```diff
@@ -0,0 +1,11 @@
+import Image from "next/image";
+
+export const LineIcon = () => {
```

Comment:
Is there any reasons to use arrow function instead of regular function?
If there is only one export, consider default export this function

Reply:
done